content/guides/authentication.md

```diff
@@ -73,15 +73,15 @@ eosc -H "Authorization: Bearer YOURTOKENHERE" -u <https://mainnet.eos.dfuse.io>
 {{< /tab >}}
 {{< /tabs >}}
 
-Don't forget to include your token in the above command with a valid JWT, retrieved using [`/v1/auth/issue`](#post-v1-auth-issue)
+Don't forget to replace the token in the above command with a valid JWT, retrieved using [`/v1/auth/issue`](#post-v1-auth-issue)
```

Comment:
Dead link, we should update it.

With Hugo, the way to do this is by using the `ref` shortcode. Example for here:

```
...retrieved using [`/v1/auth/issue`]({{< ref "/reference/eosio/rest/auth-issue-intro" >}})
```

`auth-issue-intro` being the name of the .md file you are trying to link to. Using this method will also add validation at build time that the link is not dead, protecting us against broken links when we move files, rename stuff, etc.

The format for all hugo shortcodes is `{{< shortcodeName "someoptions" >}}`